perlin.py

```diff
@@ -62,7 +62,7 @@ def get_population_number(edge: sumolib.net.edge.Edge, base: float, centre,
     :return: the scaled noise value as float in [0:1]
     """
     x, y = get_edge_pair_centroid(edge.getShape())
-    return get_perlin_noise(x=float(x), y=float(y), base=base, scale=scale, octaves=octaves) + (
+    return get_perlin_noise(x, y, base=base, scale=scale, octaves=octaves) + (
```

Comment:
In #12 there is an erratic change casting base to int, but this is not present here - yet this PR depends on #12?

Reply:
They are now in sync again!